service/switchboardapp/src/test/java/com/switchboard/app/EncoderControllerTest.java

```diff
@@ -0,0 +1,114 @@
+package com.switchboard.app;
+
+import com.switchboard.app.controller.EncoderController;
+import com.switchboard.app.dao.DeviceDaoImpl;
+import com.switchboard.app.dao.EncoderDaoImpl;
+import com.switchboard.app.domain.DeviceEntity;
+import com.switchboard.app.domain.EncoderEntity;
+import com.switchboard.app.exceptions.DeviceNotFoundException;
```

Comment:
`import com.switchboard.app.domain.DeviceEntity;` needs to change to `import com.switchboard.app.entity.DeviceEntity;`

`import com.switchboard.app.domain.EncoderEntity;` needs to change to `import com.switchboard.app.entity.EncoderEntity;`

`import com.switchboard.app.exceptions.DeviceNotFoundException;` needs to change to `com.switchboard.app.exceptions.ExceptionType;`

If not changed, we will get a compile error on merge.

Reply:
Has been updated